src/app/app.component.ts

```diff
@@ -49,11 +49,33 @@ export class AppComponent implements OnInit {
             this.routerLink = '/analyze/' + this.label;
             // this.gateway['user_key'] = this.apiData['user_key'];
             this.gateway['access_token'] = this.apiData['access_token'];
-            this.stackUrl = 'https://recommender.api.openshift.io/api/v1/stack-analyses/' + this.label;
-            console.log('=========================');
-            console.log(this.gateway);
-            console.log(this.stackUrl);
-            console.log('=========================');
+            this.gateway['config'] = this.apiData['route_config'];
+
+            // {
+            //     "route_config": {
+            //         "api_url": ""
+            //     }
+            // }
+
+            // In case of 3 scale
```

Comment:
we can remove comments

Reply:
I kept it intentionally to handle for 3 scale.

Reply:
got it!!